### DISCUSSION TITLE
Updating a tenant domain

### DISCUSSION THREAD

**Discussion by mast3rbow** [django-tenants/django-tenants#648] [Q&A]
Hi team,

This might be a really simple thing to do, and i could be totally overthinking it. But how do I use shell commands update an existing tenants domain? For example if I am moving the site to a new domain.

I'm sure it super simple, but I'm still learning Python and this seems to have to stumped me.

**Comment by PathakHarsh** [comment]
Suppose you have made a tenant with  a domain name 'tenant.localhost'

The tenant model must be registered in the public schema.Find the Id of the tenant,whose domain you want to change.

Then In the terminal run the command 
'python manage.py shell'

After opening the shell import the tenant model from your app.

Suppose your tenant model name is client,then call these commands

tenant = Client.objects.get(id="the I'd of the tenant whose domain change is required")

tenant.domain_url = ' new domain name'

tenant.save()

I hope this helps
